fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -208,106 +187,123 @@ public class Loan extends AbstractAuditableWithUTCDateTimeCustom {
     private ExternalId externalId;
 
     @ManyToOne
-    @JoinColumn(name = "client_id", nullable = true)
+    @JoinColumn(name = "client_id")
     private Client client;
 
+    @Getter
```

Comment:
We have Lombok getter setter on the class. if we would like to manually decide on which field we would like to generate one, we should remove that one. Thoughts?